src/main/resources/assets/mythicmetals/lang/de_de.json

```diff
@@ -0,0 +1,1146 @@
+
+  "smithing_template.mythicmetals.midas_folding.title": "Midasgoldschwert Folding",
+  "smithing_template.mythicmetals.midas_folding.applies_to": "Midasgoldschwerter",
+  "smithing_template.mythicmetals.midas_folding.ingredients": "Midasgoldblock",
+  "smithing_template.mythicmetals.midas_folding.base_slot_description": "Ein Midasgoldschwert hier reinlegen.",
+  "smithing_template.mythicmetals.midas_folding.additions_slot_description": "Einen Midasgoldblöcke hier reinlegen.",
+
+  "smithing_template.mythicmetals.royal_midas.title": "Royales Midasgoldschwert Upgrade",
+  "smithing_template.mythicmetals.royal_midas.applies_to": "Vergoldetes Midasgoldschwert",
+  "smithing_template.mythicmetals.royal_midas.ingredients": "Midasgoldblock",
+  "smithing_template.mythicmetals.royal_midas.base_slot_description": "Ein Vergoldetes Midasgoldschwert hier reinlegen.",
+  "smithing_template.mythicmetals.royal_midas.additions_slot_description": "Noch einen Midasgoldblock hier reinlegen.",
+
+  "smithing_template.mythicmetals.carmot.title": "Carmot Upgrade",
+  "smithing_template.mythicmetals.carmot.applies_to": "Kyberausrüstung",
+  "smithing_template.mythicmetals.carmot.ingredients": "Carmotbarren",
+  "smithing_template.mythicmetals.carmot.base_slot_description": "Eine Kyberausrüstung hier reinlegen.",
+  "smithing_template.mythicmetals.carmot.additions_slot_description": "Ein Carmotbarren hier reinlegen.",
+
+  "smithing_template.mythicmetals.legendary_banglum.title": "Legendäres Banglum Upgrade",
+  "smithing_template.mythicmetals.legendary_banglum.applies_to": "Banglumausrüstung",
+  "smithing_template.mythicmetals.legendary_banglum.ingredients": "Banglum Chunk",
+  "smithing_template.mythicmetals.legendary_banglum.base_slot_description": "Eine Banglumausrüstung hier reinlegen.",
+  "smithing_template.mythicmetals.legendary_banglum.additions_slot_description": "Ein Banglum Chunk hier reinlegen.",
+
+  "smithing_template.mythicmetals.unobtainium.title": "Unerhaltbarium Alloy Upgrade",
+  "smithing_template.mythicmetals.unobtainium.applies_to": "Netherit- oder Diamandausrüstung",
+  "smithing_template.mythicmetals.unobtainium.ingredients": "Metallurgiumbarren oder Celestiumbarren",
+  "smithing_template.mythicmetals.unobtainium.base_slot_description": "Für Metallurgium eine Netheritausrüstung hier reinlegen. Für Celestium eine Diamandausrüstung hier reinlegen.",
+  "smithing_template.mythicmetals.unobtainium.additions_slot_description": "Je nach Wahl, Metallurgium- oder Celestiumerz hier",
+
+  "attribute.name.generic.mythicmetals.fire_vulnerability": "Feueranfälligkeit",
+  "attribute.name.generic.mythicmetals.elytra_rocket_speed": "Elytra Racketengeschwindigkeit",
+  "item.spectrum.infused_beverage.tooltip.variant.morkite": "Dunkles Morkit",
+  "item.spectrum.infused_beverage.tooltip.variant.wormhole_special": "Wurmloch Spezial",
+  "effect.mythicmetals.wormhole_special": "Wurmloch Spezial",
+  "effect.mythicmetals.heat": "Heat",
```

Comment:
Untranslated. This effect is a mark from Palladium, which turns into combustion if set on fire.